spec/integration/session_spec.rb

```diff
@@ -804,7 +804,9 @@
 
         expect {
           @session.within_frame('omg') { }
-        }.to raise_error(Capybara::Poltergeist::FrameNotFound)
+        }.to raise_error { |e|
+          expect(e).to be_a(Capybara::Poltergeist::FrameNotFound).or be_a(Capybara::ElementNotFound)
```

Comment:
Because from 2.8 on Capybara handles finding the frame element and just passes it to the driver.  That means it will return with ElementNotFound before the driver is even called if the frame doesn't exist